README.md

```diff
@@ -1,21 +1,18 @@
 # conjur-api-python3
 
-Python3-based API SDK for [Conjur OSS](https://www.conjur.org/). The repo
-also includes a self-contained CLI tool (`conjur-cli`) that wraps the API
+Command-line interface and programmatic Python access to the Conjur API.
+
+The repo also includes a self-contained CLI tool (`conjur-cli`) that wraps the API
 in a simple executable script/binary.
 
 [![Test Coverage](https://api.codeclimate.com/v1/badges/d90d69a3942120b36785/test_coverage)](https://codeclimate.com/github/cyberark/conjur-api-python3/test_coverage) [![Maintainability](https://api.codeclimate.com/v1/badges/d90d69a3942120b36785/maintainability)](https://codeclimate.com/github/cyberark/conjur-api-python3/maintainability)
 
 ---
 
-### **Status**: Alpha
-
-#### **Warning: Naming and APIs are still subject to breaking changes!**
+### **Status**: GA
```

Comment:
Is GA is for SDK and CLI?
We need to make it GA only once we release it... after signing & testing...

Reply:
This is a good question. What requirements are need to be met in order for client as well to be GA level?

Reply:
1. product and UX review
2. documentation
3. test coverage
4. security review 

Basically our regular DOD
Also the link it self can help:
https://github.com/cyberark/community/blob/master/Conjur/conventions/certification-levels.md

In any case CLI is planned for GA :)

Reply:
so our plan is to take the CLI to GA but we don't have different versions for the CLI and SDK in PyPi because they are shipped together so I think
How do we usually handle this? @InbalZilberman @izgeri ? Can we promote one but not the other? How would versioning work?